### DISCUSSION TITLE
Can I stop the Zed assistant in the side panel once it starts generatign?

### DISCUSSION THREAD

**Discussion by yodaiken** [zed-industries/zed#18840] [Feature Requests]
I often find myself starting a generation, seeing it go down the wrong path, and just have to ... wait it out. Am I missing a good wait to hit "stop"?

**Comment by bennetbo** [comment]
Yes, you can run `editor: cancel` or hit `esc` to cancel the LLM response

https://github.com/user-attachments/assets/898cf61e-04af-4904-973a-e5e693a8ad35
